lib/models/room.js

```diff
@@ -415,6 +415,11 @@ function setEventMetadata(event, stateContext, toStartOfTimeline) {
 function calculateRoomName(room, userId) {
     // check for an alias, if any. for now, assume first alias is the
     // official one.
+    var mRoomName = room.currentState.getStateEvents('m.room.name', '');
```

Comment:
Nit: Please use `"` in this project. This is even done nearly everywhere in the SDK :exclamation: